dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -275,7 +271,7 @@
 
   <!-- unzip core release artifact, install plugin, then start ES -->
   <target name="start-external-cluster-with-plugin" depends="setup-workspace">
-    <install-plugin name="${project.artifactId}" file="${project.build.directory}/releases/${project.artifactId}-${project.version}.zip"/>
+    <install-plugin name="${project.artifactId}" fileUrl="file://${project.build.directory}/releases/${project.artifactId}-${project.version}.zip"/>
```

Comment:
This isn't correct. Why make this change? the call to makeurl as it is done currently in master is the only correct way. string concatenation simply will not work, if the directory contains whitespace for example, or certain other characters (it must be a true URL)